features/element-hiding.json

```diff
@@ -10,6 +10,10 @@
         {
             "domain": "duckduckgo.com",
             "reason": "https://github.com/duckduckgo/privacy-configuration/issues/1236"
+        },
+        {
+            "domain": "duck.co",
+            "reason": "[TODO: Move to GitHub issue] Element hiding is targetting the ad badge on dev instances as they are not under the duckduckgo.com domain anymore"
```

Comment:
Happy just to leave in the comment rather than the gh issue link. We ditched doing that.

Reply:
Done. Thanks